types/taiko/index.d.ts

```diff
@@ -0,0 +1,366 @@
+     */
+
+    // https://taiko.gauge.org/#dollar
+    export function $(selector: string, ...args: SearchElement[]): Selector;
+    // https://taiko.gauge.org/#image
+    export function image(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#link
+    export function link(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#listitem
+    export function listItem(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#button
+    export function button(selector: SearchElement, ...args: SearchElement[]): SearchElement;
+    // https://taiko.gauge.org/#inputfield
+    export function inputField(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#filefield
+    export function fileField(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#textbox
+    export function textBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#combobox
+    export function comboBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#dropdown
+    export function dropDown(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#checkbox
+    export function checkBox(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#radiobutton
+    export function radioButton(selector: SearchElement, ...args: SearchElement[]): ElementWrapper;
+    // https://taiko.gauge.org/#text
+    export function text(selector: string, ...args: SearchElement[]): ElementWrapper;
+
+    /**
+     * Proximity Selectors
+     */
+
+    // https://taiko.gauge.org/#toleftof
+    export function toLeftOf(selector: SearchElement | ElementWrapper): SearchElement;
+    // https://taiko.gauge.org/#torightof
+    export function toRightOf(selector: SearchElement | ElementWrapper): SearchElement;
+    // https://taiko.gauge.org/#above
+    export function above(selector: SearchElement | ElementWrapper): SearchElement;
+    // https://taiko.gauge.org/#below
+    export function below(selector: SearchElement | ElementWrapper): SearchElement;
+    // https://taiko.gauge.org/#near
+    export function near(selector: SearchElement | ElementWrapper, opts?: ProximitySelectorNearOptions): SearchElement;
+
+    /**
+     * Events
+     */
+
+    // https://taiko.gauge.org/#prompt
+    export function prompt(message: string, callback: Function): void;
+    // https://taiko.gauge.org/#confirm
+    export function confirm(message: string, callback: Function): void;
+    // https://taiko.gauge.org/#beforeunload
+    export function beforeunload(message: string, callback: Function): void;
+
+    /**
+     * Helpers
+     */
+
+    // https://taiko.gauge.org/#evaluate
+    export function evaluate(
```

Comment:
can take EventOptions too

Reply:
that should already be the case, since the NavigationOptions extend EventOptions like this:
```typescript
export interface NavigationOptions extends BasicNavigationOptions, EventOptions {}
```